smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/middleware/MiddlewareExecutionGenerator.kt

```diff
@@ -119,6 +119,19 @@ class MiddlewareExecutionGenerator(
         httpProtocolCustomizable.renderContextAttributes(ctx, writer, serviceShape, op)
     }
 
+    private fun resolveHttpMethod(op: OperationShape): String {
+        val httpMethod = when (executionContext) {
+            MiddlewareRenderableExecutionContext.PRESIGNER_POLLY_GET_REQUEST -> {
```

Comment:
is this maybe a case for a sectino writer? feels weird to have something so aws specific in smithy-swift since we know well have to refactor this out at some point